tests/jobs/test_triggerer_job.py

```diff
@@ -110,6 +112,9 @@ def create_trigger_in_db(session, trigger, operator=None):
     return dag_model, run, trigger_orm, task_instance
 
 
+# Quarantined because this test is failing the first time it is run. The second time succeeds
+# Seems to be connected with SOME race condition and needs to be investigated
+@pytest.mark.quarantined
```

Comment:
I have to move this test to quarantine - it behaves really strange - first time you run it in breeze it fails (even in main when I run it indivdually) and second and all next runs it succeeds. 

I will open an issue about it shortly and likely will need help of @hussein-awala and @dstandish to unquarantine this one.

Reply:
I checked the failure, and found that it's because we received an empty stdout when we reset the db (I still do not know why), so I wonder if we can just remove:
```
assert "test_dag/test_run/sensitive_arg_task/-1/1 (ID 1) starting" in stdout
```
and keep
```
assert "some_password" not in stdout
```
instead of quarantine it 🤔

The goal of this test is to ensure that the sensitive data are not logged, and this change should be okay, wdyt?

Reply:
Well. not really.. There is some reason why we get empty string but I think the test has a bit different assertion:

WHEN we receive `test_dag/test_run/sensitive_arg_task/-1/1 (ID 1) starting` make sure that "some_password" is not there (it would be there if not masked). So I think if we just test for "some password" - we at most checj that we have not "yet" received sensitive information :D

Reply:
Captured all details for that one in https://github.com/apache/airflow/issues/35254